rskj-core/src/test/java/co/rsk/peg/bitcoin/BitcoinTestUtils.java

```diff
@@ -154,4 +155,23 @@ private static void signLegacyTransactionInputFromP2shMultiSig(BtcTransaction tr
             input.setScriptSig(inputScriptSig);
         }
     }
+
+    public static List<Sha256Hash> generateTransactionInputsSigHashes(BtcTransaction btcTx) {
+        List<Sha256Hash> sigHashes = new ArrayList<>();
+        List<TransactionInput> inputs = btcTx.getInputs();
+        for (TransactionInput input : inputs) {
+            Sha256Hash sigHash = generateSigHashForP2SHTransactionInput(btcTx, inputs.indexOf(input));
+            sigHashes.add(sigHash);
+        }
+        return sigHashes;
+    }
+
+    public static List<byte[]> generateSignerEncodedSignatures(BtcECKey signingKey, List<Sha256Hash> sigHashes) {
```

Comment:
```suggestion
    public static List<byte[]> signInputSigHashes(BtcECKey signingKey, List<Sha256Hash> sigHashes) {
```

I would prefer this name. Wdyt?

Reply:
but is not signing anything 🤔 is generating the encoded signatures

Reply:
mmmm

Reply:
This process(sign a transaction) is called and known as "signing" not `generating`.

Reply:
but im not just signing, i want the list of the encoded signatures.